newPassword.php

```diff
@@ -0,0 +1,40 @@
+<?php
+session_start();
+$email = $_POST["email"];
+$dob = $_POST["dob"];
+
+include 'connection.php';
+$sql = "SELECT * FROM users WHERE email=$email";
+$result = $connection->query($sql);
+
+if ($row = $result->fetch_assoc) {
+  if ($dob == $row["dob"]) {
+    $_SESSION["forgotPasswordEmail"] = $email;
+    header('Location: ./index.php?alert=Your Password has been reset');
```

Comment:
Yeah, I was not really sure what you were trying to with your previous code. Now, I understood what you are trying to do. Redirect is not necessary.